assign_policy.yml

```diff
@@ -0,0 +1,17 @@
+---
+- hosts: hosts
+
+  tasks:
+  - name: Assign policy on an OpenShift provider in ManageIQ
+    manageiq_policy:
+      entity: '{{ entity }}'
+      entity_name: '{{ entity_name }}'
+      resource: '{{ resource }}'
+      resource_name: '{{ resource_name }}'
+      action: '{{ action }}'
```

Comment:
@kwoodson The policy module is meant to cover all policy and policy profiles related actions:
* assign, unassign, resolve - currently supported
* CRUD actions on policy/policy profile, including actions and conditions - not yet supported in manageiq API

We need to decide with which approach we want to continue here:
a. keep this module only for assign, unassign and resolve (and make the changes you asked with state)
b. add all CRUD missing actions when possible.

@simon3z, @cben, @enoodle what is your opinion?